symbulate/random_variables.py

```diff
@@ -23,6 +36,13 @@ def check_same_probSpace(self, other):
             self.probSpace.check_same(other.probSpace)
 
     def apply(self, function):
+        """
+        Args:
+            function: function being used on results (addition, subtractions, etc.)
```

Comment:
Suggestion: replace by 

function: function to apply to the random variable (e.g., log, sqrt, exp)